composer.json

```diff
@@ -11,16 +11,16 @@
         }
     ],
     "require": {
-        "php": "^7.2.5",
-        "guzzlehttp/guzzle": "~6.0",
-        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0 || ^8.0",
-        "illuminate/support": "~5.5 || ~6.0 || ~7.0 || ^8.0"
+        "php": "^7.3",
+        "guzzlehttp/guzzle": "~7.0",
+        "illuminate/notifications": "~6.0 || ~7.0 || ^8.0",
```

Comment:
If we are dropping support for L6, maybe the `~6.0` need to be removed as well?

Reply:
Oops, you're right. Fixed !